tests/test_urban_nature_access.py

```diff
@@ -246,3 +320,16 @@ def test_model(self):
         numpy.testing.assert_allclose(
             aligned_lulc_raster_info['bounding_box'],
             aligned_population_raster_info['bounding_box'])
+
+        # Check that we're getting the appropriate summary values in the admin
+        # units vector.
+        admin_vector_path = os.path.join(
+            args['workspace_dir'], 'output',
+            f"admin_units_{args['results_suffix']}.gpkg")
+        admin_vector = gdal.OpenEx(admin_vector_path)
+        admin_layer = admin_vector.GetLayer()
+        self.assertEqual(admin_layer.GetFeatureCount(), 1)
+
+        admin_feature = admin_layer.GetFeature(1)
+        numpy.testing.assert_allclose(
+            admin_feature.GetField('average_greenspace_budget'), -17.9078)
```

Comment:
Can we make a comment about this hard coded value. Was it confirmed via the eye test?

Reply:
For sure!  d2180721